src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -207,17 +207,6 @@ public enum ErrorCode {
   TABLE_COLUMN_UNKNOWN("Column unknown");
 
   private final String message;
-  private final boolean extendError =
```

Comment:
Access from constructor of EVERY enum constant is pretty wasteful to begin with, but more importantly caused odd initialization failure from any Integration Test that tried to refer to `ErrorCode` enums.
Hence moved to be done lazily when needed, not eagerly before everything else.